data/index.ts

```diff
@@ -109,5 +110,10 @@ export default {
   generateBenefitId,
   fn: generate,
   generate: (jam) =>
-    generate(jam.collections, additionalBenefitAttributes, aidesVeloGenerator),
+    generate(
+      jam.collections,
+      additionalBenefitAttributes,
+      aidesVeloGenerator,
+      build
```

Comment:
```suggestion
      buildFSL
```

pour plus de lisibilité future (a changer dans l'import aussi) du coup (et peut-être ailleurs)